### ISSUE TITLE
Extension info tab: reload images everytime

### ISSUE THREAD

**Issue by samzzz** [microsoft/vscode#36310]
- VSCode Version: Code 1.17.1 (1e9d36539b0ae51ac09b9d4673ebea4e447e5353, 2017-10-10T14:24:50.851Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: none
---
The Extension info tab (Details) reload images every-time I come-back to the extension's tab after activating another tab. Please use cache to avoid this. This reload append even when I stay on the extension tab and I switch to another section: Contributions, ChangeLog, Dependencies, then I come-back to Details.
Thanks in advance and regards,

Steps to Reproduce:
1.Open extension panel and search a extension.
2.Select an extension to display its info in a new tab.
3.Switch to another tab (file.html)
4.Comeback to the extension's tab an you can notice the images are reloaded
5.Repeat 3 and 4.

**Comment by joaomoreno** [comment]
@mjbvz Can we do some caching in the webview?

**Comment by TheColorRed** [comment]
In my extension, you type into a textbox, is it possible to add what was type to the url with a hash like so?

    laravel-artisan://prefix/route-list#my-search-term

**Comment by Thaurin** [comment]
This happens with the Release Notes tab as well. The pause before the tab is reloading destroys the user experience. If this could be improved upon, that would be awesome.

**Comment by mjbvz** [comment]
I'm closing this as out of scope due to performance reasons. We could keep the extension page around like a background tab like in a browser, but that would mean we would have to keep it in memory. This is not good use of memory for an editor in my opinion

**Comment by KeithSilcock** [comment]
How about a setting? It's tedious when I have to reference these pages often enough that reloading the entire page and then scrolling to where I left off every time is more of a hassle than having less memory.

**Comment by Hadakajime** [comment]
Opinion: kinda disturbed by the hubris of closing this without giving any consideration for the impact on UX.

Case: I am setting up VSCode and went to install the Terminals Manager Extension. To complete the install, I had to edit the extension config. I did this by running the command to open the json config in a new tab, then attempted to switch between that tab and the extension's marketplace page tab to reference the config guide.

Result: Horrible UX from the marketplace tab constantly reloading from the slightest and most miniscule tab switch. What is more valuable - a little bit of RAM, or my time and patience?
